bassist/bassist/parser/log_file/debs_stdout.py

```diff
@@ -2,10 +2,11 @@
 # See the file LICENSE for copying permission.
 from . import common
 from ...systems import package
-from results import store_debs
-from run_sql import execute_sql
+
+debs = []
```

Comment:
This commit should be part of the PR: Added check_exists functions in DB    2f37008

 I just added it now